src/secure_random.cr

```diff
@@ -1,20 +1,64 @@
 require "base64"
 require "openssl/lib_crypto"
 
+# The SecureRandom module is an interface for creating secure random values in different formats, 
+# including Base64 string, hexadecimal string, uuid and others. It uses LibCrypto(OpenSSL) as the
+# source for generation.
+#
+# For example:
+# ```crystal
+# SecureRandom.base64 #=> LIa9s/zWzJx49m/9zDX+VQ==
```

Comment:
`#=>` should list valid Crystal literals / `.inspect` output, so these are missing quotes `"`